src/docs/error-handling.dox

```diff
@@ -6,19 +6,33 @@ positive error codes are standard error codes as described for
 POSIX-like systems (for example, \c EINVAL or \c EBUSY), negative error
 codes are WiredTiger-specific (for example, \c WT_ROLLBACK).
 
-WiredTiger-specific error codes always appear in the -31,800 to -31,999
-range, inclusive.
+@section error_ebusy EBUSY errors
 
 WiredTiger returns \c EBUSY for operations requiring exclusive access, when
-an object is not available for exclusive access. For example, the
-WT_SESSION::drop or WT_SESSION::verify methods will fail if the object
-has open cursors. Note that internal WiredTiger threads may temporarily
-open cursors on objects (for example, threads performing operations like
-statistics logging), and operations may temporarily fail and return \c EBUSY
-when there are no application cursors open on the object.
-
-The following is a complete list of the WiredTiger-specific return
-values:
+an object is not available for exclusive access. These operations include
+WT_SESSION::alter, WT_SESSION::drop, WT_SESSION::rename, WT_SESSION::salvage,
+WT_SESSION::upgrade and WT_SESSION::verify methods, all of which will
+return \c EBUSY and fail if there are open cursors on an object. Internal
```

Comment:
"open cursors on THE object". (or on "THE TARGET object"?). 

I.e., we don't return EBUSY if some other object has open cursors.

Reply:
Fixed in c9b76eef7.